python/cog/server/http.py

```diff
@@ -74,16 +86,20 @@ def create_app(
     InputType = get_input_type(predictor)
     OutputType = get_output_type(predictor)
 
-    PredictionRequest = schema.PredictionRequest.with_types(input_type=InputType)
+    class PredictionRequest(schema.PredictionRequest.with_types(input_type=InputType)):
+        pass
```

Comment:
why?

Reply:
tldr this allows typingchecking PredictionRequest usage by pyright (still not mypy, though)